### PR TITLE
Import eosio key automatically when a wallet is created - #2008

### PR DESCRIPTION
**Pull request by zorba80**

### CODE REVIEW

File: plugins/wallet_plugin/wallet_manager.cpp [hunk 1]
```diff
@@ -40,16 +40,17 @@ std::string wallet_manager::create(const std::string& name) {
...
    }
-
+   
    wallet_data d;
    auto wallet = make_unique<wallet_api>(d);
    wallet->set_password(password);
    wallet->set_wallet_filename(wallet_filename.string());
    wallet->unlock(password);
-   wallet->save_wallet_file();
+   static const std::string eosio_wif_key("5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3");
```
Comment (heifner) [inline plugins/wallet_plugin/wallet_manager.cpp]: What do you think of making this a defaulted argument to the wallet plug so a user can specify a different key without re-compiling?

Comment (wanderingbort) [reply plugins/wallet_plugin/wallet_manager.cpp]: I like this idea

Comment (zorba80) [reply plugins/wallet_plugin/wallet_manager.cpp]: Added that

File: plugins/wallet_plugin/wallet_manager.cpp [hunk 2]
```diff
@@ -40,16 +40,16 @@ std::string wallet_manager::create(const std::string& name) {
...
    }
-
+   
    wallet_data d;
    auto wallet = make_unique<wallet_api>(d);
    wallet->set_password(password);
    wallet->set_wallet_filename(wallet_filename.string());
    wallet->unlock(password);
-   wallet->save_wallet_file();
+   wallet->import_key(eosio_key);
```
Comment (wanderingbort) [inline plugins/wallet_plugin/wallet_manager.cpp]: was `save_wallet_file()` no longer necessary here?

Comment (wanderingbort) [reply plugins/wallet_plugin/wallet_manager.cpp]: nevermind I see that is handled in the wallet api for import_key

**Pull request merged**